web/thesauruses/python/3/queues_stacks.json

```diff
@@ -0,0 +1,388 @@
+{
+    "meta": {
+      "language": "python",
+      "language_name": "Python",
+      "structure": "queues_stacks",
+      "language_version": "3"
+    },
+    "concepts": {
+      "simple_queue_import_statement": {
+        "name": "Import statement to add simple queue",
+        "not-implemented": "true"
+      },
+      "simple_queue_data_type": {
+        "name": "Queue data type name",
+        "code": [
+          "list"
```

Comment:
You use a list, which _can_ work for this, but specifically there's [queue.Queue](https://docs.python.org/3/library/queue.html) which seems to be an official queue in Python with fast enqueues and dequeues. It's in the same library as dequeues and PriorityQueues, which you use below. Since lists are described in the `lists.json` file, this should probably be for the built-in queue objects.